demos/coreHTTP/http_demo_s3_download_multithreaded.c

```diff
@@ -393,7 +434,7 @@ static BaseType_t prvDownloadS3ObjectFile( const char * pcHost,
                                            QueueHandle_t xResponseQueue )
 {
     BaseType_t xStatus = pdPASS;
-    size_t xRequestCount = 0;
+    size_t xResponseCount = 0;
```

Comment:
Can we use a variable to take a pdTRUE or pdFALSE instead of a count? As I understand, we will always expecting only a response at a time.